package.json

```diff
@@ -3,14 +3,17 @@
   "version": "0.0.1",
   "description": "Products service for Project Atelier API rebuild",
   "main": "./server/index.js",
+  "scripts": {
+    "server-dev": "npx nodemon --watch server server/index.js"
+  },
   "repository": "https://github.com/Team-Mandalore/SDC-Products.git",
   "author": "sbirvin1s <sbirvin1s@gmail.com>",
-  "license": "MIT",
```

Comment:
Question:  Why did you change it to unlicensed?  I'm just trying to learn more.

Reply:
I changed it to unlicensed because I wasn't sure what was included in the "MIT" license. That was added by yarn when I initiated the package-json and I Wanted to make sure I wasn't promising anything with the license that I wasn't aware of.

Mostly me being paranoid 😅

Reply:
Looks like it is specific guidance on how the code can be used by others. This [Wikipedia article](https://en.wikipedia.org/wiki/MIT_License) has some interesting information on it 

Here is an excerpt from Wikipedia on it as well

>Copyright (c) <year> <copyright holders>
>
>Permission is hereby granted, free of charge, to any person obtaining a copy
of this software and associated documentation files (the "Software"), to deal
in the Software without restriction, including without limitation the rights
to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
copies of the Software, and to permit persons to whom the Software is
furnished to do so, subject to the following conditions:
>
>The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.
>
>THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
SOFTWARE.